cli/pcluster/cfnconfig.py

```diff
@@ -528,6 +532,23 @@ def __init_cluster_parameters(self):
             except configparser.NoOptionError:
                 pass
 
+    def __init_ec2_policies(self):
+        try:
+            __temp__ = self.__config.get(self.__cluster_section, "additional_iam_policies")
+            if not __temp__:
+                self.__fail(
+                    "%s defined but not set in [%s] section" % ("additional_iam_policies", self.__cluster_section)
+                )
+            iam_policies = list(map(lambda x: x.strip(), __temp__.split(",")))
+            if self.parameters.get("Scheduler") == "awsbatch":
+                aws_batch_iam_policy = "arn:{0}:iam::aws:policy/AWSBatchFullAccess".format(get_partition(self.region))
+                if not (aws_batch_iam_policy in iam_policies):
```

Comment:
not sure if this is correct. It probably needs to be aws_batch_iam_policy not in iam_policies